mrtt-ui/src/views/Auth/LoginForm.js

```diff
@@ -75,29 +76,36 @@ const LoginForm = ({ isUserNew }) => {
     navigate(-1)
   }
 
-  const handleSignUpOnClick = () => {
-    navigate('/auth/signup')
-  }
-
   const form = (
     <MainFormDiv>
       <PagePadding>
-        <PageTitle>Login</PageTitle>
+        <PageTitle>{pageLanguage.title}</PageTitle>
         {isUserNew ? (
           <Alert variant='outlined' severity='success'>
             {language.success.signup}
           </Alert>
         ) : null}
+        <RowFlexEnd>
```

Comment:
When I put this link under the submit button it was aligned weirdly and looked strange. Opted to put it at the top right of the login form to save time by doing some 'dev ux'.

Reply:
I think its fine for now!